src/lib/Security/AuthorizationHeaderRESTRequestMatcher.php

```diff
@@ -6,7 +6,7 @@
  */
 declare(strict_types=1);
 
-namespace Ibexa\Contracts\Rest\Security;
+namespace Ibexa\Rest\Security;
```

Comment:
First of all this class shouldn't be replaced with any custom implementation as it's vital for REST authentication. I also mark it as internal.
> Or is it because you'd like to keep this consistent with the previous similar change?

Yes, however it's not an authenticator class. Nevertheless - to me at this point we shouldn't aim to provide any extensibility for authorization process other than built-in events coming from Symfony (e.g. InteractiveLoginEvent) for safety reason.